Tests/LibWeb/Text/input/HTML/Window-find-mutations.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<script src="../include.js"></script>
+<div>foobar</div>
+<script>
+test(() => {
+    // Select 'foobar'.
+    let divElm = document.querySelector('div');
+    window.find('foobar');
+    let range = getSelection().getRangeAt(0);
+    println(`Selection: ${range.startContainer} ${range.startOffset} - ${range.endContainer} ${range.endOffset}`);
+
+    // Remove 'bar'.
+    divElm.childNodes[0].deleteData(3, 3);
+    println(`Selection: ${range.startContainer} ${range.startOffset} - ${range.endContainer} ${range.endOffset}`);
+
+    // Try to find 'bar'.
+    window.find('bar');
+
+    // This should now fail.
+    try {
+        getSelection().getRangeAt(0);
+        println('FAIL');
+    } catch (e) {
+        println(`Expected exception: ${e}`);
```

Comment:
I'm not sure what the behavior is supposed to be, but neither Chrome nor Firefox hit this exception.